modules/web-console/frontend/app/components/page-configure/components/modal-import-models/template.tpl.pug

```diff
@@ -225,7 +225,11 @@ mixin td-ellipses-lbl(w, lbl)
                                 tipOpts
                             })
 
-            .modal-footer
+
+            .modal-body(ng-if='!($ctrl.agentIsAvailable$ | async: this)')
```

Comment:
You can use ng-if-start and ng-if-end directives to hide multiple sibling nodes.

Reply:
Good point. Changed.